pypeit/core/convert_DEIMOSsavfiles.py

```diff
@@ -0,0 +1,33 @@
+from scipy.io import readsav
```

Comment:
I doubt we'll need this function again, but we might want to move it to `pypeit.io`.  I would leave it for now.